include/openthread/nat64.h

```diff
@@ -200,6 +315,59 @@ bool otIp4IsAddressEqual(const otIp4Address *aFirst, const otIp4Address *aSecond
  */
 void otIp4ExtractFromIp6Address(uint8_t aPrefixLength, const otIp6Address *aIp6Address, otIp4Address *aIp4Address);
 
+/**
+ * Gets the configured CIDR in the NAT64 translator.
+ *
+ * @param[in]  aInstance         A pointer to an OpenThread instance.
+ * @param[out] aCidr             A pointer to an otIp4Cidr. Where the configured CIDR will be filled.
+ *
+ */
+otError otNat64GetConfiguredCidr(otInstance *aInstance, otIp4Cidr *aCidr);
+
+/**
+ * Gets the configured CIDR in the NAT64 translator.
+ *
+ * @param[in]  aInstance         A pointer to an OpenThread instance.
+ * @param[out] aPrefix           A pointer to an otIp6Prefix. Where the configured NAT64 prefix will be filled.
+ *
+ */
+otError otNat64GetConfiguredPrefix(otInstance *aInstance, otIp6Prefix *aPrefix);
```

Comment:
nit: move up the two otNat64* functions to be together with other otNat64* functions